mlflow/cli/eval_utils.py

```diff
@@ -0,0 +1,188 @@
+    builtin_scorer_map = {scorer.__class__.__name__: scorer for scorer in builtin_scorers}
+
+    for scorer_name in scorer_names:
+        # Check if it's a built-in scorer
+        if scorer_name in builtin_scorer_map:
+            resolved_scorers.append(builtin_scorer_map[scorer_name])
+        else:
+            # Try to get it as a registered scorer
+            try:
+                registered_scorer = get_scorer(name=scorer_name, experiment_id=experiment_id)
+                resolved_scorers.append(registered_scorer)
+            except MlflowException:
+                available_builtin = ", ".join(builtin_scorer_map.keys())
+                raise click.UsageError(
+                    f"Scorer '{scorer_name}' not found. "
+                    f"Available built-in scorers: {available_builtin}. "
+                    f"Or register a custom scorer first."
+                )
+
+    if not resolved_scorers:
+        raise click.UsageError("No valid scorers specified")
+
+    return resolved_scorers
+
+
+def extract_assessments_from_results(
+    results_df: pd.DataFrame, evaluation_run_id: str
+) -> list[dict[str, Any]]:
+    """
+    Extract assessments from evaluation results DataFrame.
+
+    The evaluate() function returns results with a DataFrame that contains
+    an 'assessments' column. Each row has a list of assessment dictionaries
+    with metadata including 'mlflow.assessment.sourceRunId' that we use to
+    filter assessments from this specific evaluation run.
+
+    Args:
+        results_df: DataFrame from evaluate() results containing assessments column
+        evaluation_run_id: The MLflow run ID from the evaluation that generated the assessments
+
+    Returns:
+        List of dictionaries with trace_id and assessments
+    """
+    output_data = []
+
+    for _, row in results_df.iterrows():
+        trace_id = row["trace_id"]
+        trace_result = {"trace_id": trace_id, "assessments": []}
+
+        assessments_list = row.get("assessments", [])
+
+        if assessments_list:
+            for assessment_dict in assessments_list:
+                # Check if this assessment is from our evaluation run
+                metadata = assessment_dict.get("metadata", {})
+                source_run_id = metadata.get("mlflow.assessment.sourceRunId")
+
+                if source_run_id == evaluation_run_id:
+                    result_dict = {
+                        "assessment_name": assessment_dict.get("assessment_name"),
+                        "result": None,
+                        "rationale": None,
+                    }
+
+                    # Extract feedback value
+                    feedback = assessment_dict.get("feedback")
+                    if feedback and isinstance(feedback, dict):
+                        result_dict["result"] = feedback.get("value")
+
+                    # Extract rationale
+                    rationale = assessment_dict.get("rationale")
+                    if rationale:
```

Comment:
```suggestion
                    if rationale := assessment_dict.get("rationale")
```

Can we use the walrus operator for each of these field extraction sections and remove the inline comments, since they're somewhat self-explanatory?